src/dndbeyond/content-scripts/character.js

```diff
@@ -857,24 +857,97 @@ function displayAction(paneClass) {
     });
 }
 
+function handleCustomText(paneClass) {
+    // Function Objective: Search for and handle block of VTT Message text...
+    //...in pane Notes or Description (Priority descending in that order.)
+    const regexp = /```\[([^\]]*)\]\s*((\S|\s)*?)\s*```/g;
+    const rollOrderTypes = ["before", "after", "replace"];//(relative to normal roll msg)
+    const defaultRollFunc = function () {};
+    let rollOrder = rollOrderTypes[0];
+    let rollFunc = defaultRollFunc;
+    
+    let pane = $("."+paneClass);
+    let elemText;
+    let matchedText;
+    
+    // Search for msg text in "Notes" section, if it exists
+    //Notes: Iterate through classes in pane: "ddbc-property-list__property" that contain the text "Notes:"
+    //  If exists: load text from "ddbc-property-list__property-content" (Last child of "ddbc-property-list__property")
+    let msgElement = pane.find(".ddbc-property-list__property:contains('Notes:')").get(0)
+    if (msgElement!=undefined && msgElement!=null) {
```

Comment:
can just be `if (msgElement)`
Also, please stay consistent with the rest of the coding style, where I would put spaces in between all operators.